src/main/java/ch/njol/skript/effects/EffRemoveResourcePack.java

```diff
@@ -0,0 +1,90 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Utils;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+
+import java.util.UUID;
+
+@Name("Remove Resource Pack")
+@Description({
+	"Remove all resource packs or a resource pack by uuid sent by the server.",
+	"You can only choose to remove all or by uuid, not both",
+	"",
+	"You can allow up to 32 alphanumeric characters in the uuid."
+})
+@Examples({
+	"remove all resource packs from all players",
+	"remove resource pack with the uuid \"1\" from all players"
+})
+@Since("INSERT VERSION")
+public class EffRemoveResourcePack extends Effect {
+
+	static {
+		Skript.registerEffect(EffRemoveResourcePack.class, "remove [:all] resource pack[s] [with [the] [uu]id %-string%] from %players%");
```

Comment:
what happens if i do `remove resource pack from player`?

also, it would be easier to have it be two different patterns.